fastify.js

```diff
@@ -37,6 +39,14 @@ function build (options) {
     logger = loggerUtils.createLogger(options.logger)
   }
 
+  var ajv
+  var defaultAjvOpts = { coerceTypes: true }
+  if (options.ajv) {
+    ajv = new Ajv(Object.assign(defaultAjvOpts, options.ajv))
+  } else {
+    ajv = new Ajv(defaultAjvOpts)
+  }
```

Comment:
This could be simply:
```js
const ajv = new Ajv(Object.assign({ coerceTypes: true }, options.ajv)
```

Reply:
What do you mean?

Reply:
That you don't need that `if else` statement, is enough the single line that I write above :)

Reply:
Right 👍